app/services/vot/component_value.rb

```diff
@@ -1,5 +1,31 @@
 # frozen_string_literal: true
 
 module Vot
-  ComponentValue = Data.define(:name, :description, :implied_component_values, :requirements).freeze
+  ComponentValue = Data.define(
+    :name, :description, :implied_component_values, :requirements
+  ) do
+    # @param name [String] - unique identifier
+    # @param description [String]
+    # @param implied_component_values [Array<String>]
+    # @param requirements [Array<Symbol>] see Vot::REQUIREMENTS
+    def initialize(name:, description:, implied_component_values: [], requirements: [])
```

Comment:
Is the only real change here adding default values for these (and perhaps some documentation)? If so, does that makes sense as a separate pull request so it does not distract from the crux of the change here?